tox.ini

```
@@ -24,7 +24,7 @@ deps = flake8
 skip_install = true
 
 [testenv:isort]
-commands = isort --recursive --check-only --diff
+commands = isort --check-only --diff
```

```suggestion
commands = isort --check-only --diff .
```

isort now requires a path